docs/actions/custom.md

```diff
@@ -17,9 +17,65 @@ There are two ways to create a custom action:
 1. Click the **+ New** > **Action**. When you save your action, you'll be prompted to associate that action with a model. (NOTE: the **Action** option will only show up in the **+ New** menu if you've first created, or have access to, a [model](../data-modeling/models.md) in Metabase.)
 2. Via the model detail page: from a model, click on the **info** button in the upper right. In the upper right of the sidebar, click **Model detail** > **Actions** > **New action**.
 
-In the action editor, you can write your own code to create an action, like writing an action that would only update a subset of the columns in a table.
+In the action editor, you can write your own code to create an action, like writing an action that would only update a subset of the columns in a table. See [Example actions](#example-custom-actions).
 
-## Example `UPDATE` action
+## Field types for action variables
+
+For each {% raw %}{{ variable }}{% endraw %} that you set in an action, you'll need to set the field type.
+
+Each of these variable field types present different options. Click on the gear icon to change options.
+
+If you don't require a variable, you can optionally specify a default value for Metabase to use in cases where people don't fill out the field. In the SQL code, remember to enclose any optional variables and commas in brackets, like `{% raw %}[[, column = {{ optional_variable }}]] {% endraw %}`.
+
+You can include placeholder text for all fields in the action form.
+
+**Text**
+
+- Text
+- Long text
+- Dropdown
+- Inline select
+
+**Number**
+
+- Number
+- Dropdown
+- Inline select
+
+**Date**
+
+- Date
+- Date + Time
+
+For both **Dropdown** and **Inline select**, you can specify a list of options to present on the form, with each option on its own line.
+
+![Dropdown select](./images/dropdown.png)
+
+## Appearance
+
+The appearance tab in the action editor modal will display a preview of the variable's form element. In the image below, we've clicked on the variable'  **gears** and set the variable to use a text > dropdown list. The appearance section gives a preview for what the form element would look like:
+
+![Appearance gives a preview of the form element](./images/appearance.png)
+
+## Action settings
+
+From the model detail page, click on the **...** next to the action. Once in the action editor, click on the **gear** icon to bring up the action settings.
```

Comment:
```suggestion
From the model detail page, click on the **three dot menu** (...) next to the action. Once in the action editor, click on the **gear** icon to bring up the action settings.
```

Tiny thing: I've been writing out "three-dot menu" because ... makes me feel there's something being omitted in the middle of the sentence 🤷‍♀️